static/app/views/performance/transactionSummary/transactionReplays/useReplaysFromTransaction.tsx

```diff
@@ -93,7 +120,7 @@ async function fetchReplayIds({
       eventView.getEventsAPIPayload(location)
     );
 
-    return data.data.map(record => String(record.replayId));
+    return data;
```

Comment:
returning `data.data` might help with the types at the callsites.

either way, renaming the func is needed.